### PR TITLE
Fix various typos in Wrapping/

### PR DESCRIPTION
**Pull request by luzpaz**
Fix some user and non-user facing code typos

Signed-off-by: luz paz <luzpaz@github.com>

### CODE REVIEW

File: Wrapping/Common/mitk_swig_common.i [hunk 1]
```diff
@@ -7,7 +7,7 @@
 // 511 : Can't use keyword arguments with overloaded functions
 #pragma SWIG nowarn=302,362,503,509,511
 
-// Splitted the information about the addition files into sub-files:
+// Split the information about the addition files into sub-files:
```
Comment (luzpaz) [inline Wrapping/Common/mitk_swig_common.i:10]: Probably should be `additional`, right?

**Comment by luzpaz** [comment]
bump

**Comment by luzpaz** [comment]
Hi @kislinsk, do you mind reviewing this?

File: Wrapping/Common/mitk_swig_common.i (hunk 1 above)
Comment (kislinsk) [reply Wrapping/Common/mitk_swig_common.i:10]: Right.

Comment (luzpaz) [reply Wrapping/Common/mitk_swig_common.i]: Done 32b1dfb

**Comment by kislinsk** [comment]
Thanks!

**Pull request merged**